nats-spring-samples/multi-connect-sample/src/main/java/io/nats/spring/MultiConnectTransformer.java

```diff
@@ -16,33 +16,38 @@
 
 package io.nats.spring;
 
-import org.springframework.cloud.stream.annotation.EnableBinding;
-import org.springframework.cloud.stream.annotation.StreamListener;
-import org.springframework.cloud.stream.messaging.Processor;
-import org.springframework.messaging.handler.annotation.SendTo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
 
 import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
-@EnableBinding(Processor.class)
+@Component
 public class MultiConnectTransformer {
-    @StreamListener(Processor.INPUT)
-    @SendTo(Processor.OUTPUT)
-    public Object transform(Object message) {
-        if (message instanceof byte[]) {
-            String value = new String((byte[]) message, StandardCharsets.UTF_8);
-            StringBuilder reverse = new StringBuilder();
-            for (int i = value.length() - 1; i >= 0; i--) {
-                reverse.append(value.charAt(i));
-            }
-            message = reverse.toString().getBytes(StandardCharsets.UTF_8);
-        } else if (message instanceof String) {
-            String value = (String) message;
-            StringBuilder reverse = new StringBuilder();
-            for (int i = value.length() - 1; i >= 0; i--) {
-                reverse.append(value.charAt(i));
+    private static final Logger LOG = LoggerFactory.getLogger(MultiConnectTransformer.class);
```

Comment:
I bereave we need to get rid of logging things before merge ;)

Reply:
@YunaBraska So are we thinking on getting rid of logging from samples projects?

Reply:
ah thats a sample, then never mind :D

Reply:
Cool 😄